test/e2e/framework/service_util.go

```diff
@@ -60,7 +60,7 @@ const (
 
 	// LoadBalancerLagTimeoutDefault is the maximum time a load balancer is allowed to
 	// not respond after creation.
-	LoadBalancerLagTimeoutDefault = 2 * time.Minute
+	LoadBalancerLagTimeoutDefault = 3 * time.Minute
```

Comment:
In test/e2e/network/service.go, the test uses AWS specific timeout(LoadBalancerLagTimeoutAWS, 10mins) instead of this default timeout like https://github.com/kubernetes/kubernetes/blob/25a701db68390bd5f5323df2a17d8dd3a68af855/test/e2e/network/service.go#L526

So I think the upgrade also should do instead of increasing default timeout.

Reply:
sgtm